pkg/storaged/side-panel.jsx

```diff
@@ -113,7 +113,7 @@ class SidePanelRow extends React.Component {
                 return;
 
             // only consider enter button for keyboard events
-            if (event.type === 'keypress' && event.key !== "Enter")
+            if (event.type === 'KeyDown' && event.key !== "Enter")
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19083-20230710-104935-c8068a06-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)